Marlin/src/HAL/STM32/HAL.h

```diff
@@ -43,82 +43,67 @@
 // ------------------------
 // Defines
 // ------------------------
+#define _MSERIAL(X) MSerial##X
+#define MSERIAL(X) _MSERIAL(X)
```

Comment:
Good touch. I tend to forget that this technique can be used inside a `#define`. Now I'm curious whether the value of defines like `MYSERIAL0` are fully expanded when printed by `g++ -dM`. If so, that could be useful….